### ISSUE TITLE
Unable to open spells

### ISSUE THREAD

**Issue by BuzzardB** [vttred/ose#246]
### What happened?

Reproduction steps:
1. Create a new spell and name it
2. Attempt to open it to fill out information, nothing happens.

### What is the expected behaviour?

Spells should open.

### Relevant Errors and Warnings

```shell
foundry.js:747 Error: Parse error on line 60:
...=editable async=true}      </div>    <
-----------------------^
Expecting 'CLOSE_RAW_BLOCK', 'CLOSE', 'CLOSE_UNESCAPED', 'OPEN_SEXPR', 'CLOSE_SEXPR', 'ID', 'OPEN_BLOCK_PARAMS', 'STRING', 'NUMBER', 'BOOLEAN', 'UNDEFINED', 'NULL', 'DATA', got 'INVALID'
[No packages detected]
    at a.parseError (handlebars.min.js:28)
    at a.parse (handlebars.min.js:28)
    at d (handlebars.min.js:27)
    at d.e [as parse] (handlebars.min.js:27)
    at d (handlebars.min.js:28)
    at e (handlebars.min.js:28)
    at renderTemplate (foundry.js:6906)
    at async OseItemSheet._renderInner (foundry.js:5230)
    at async OseItemSheet._renderInner (foundry.js:5791)
    at async OseItemSheet._render (foundry.js:5073)
    at async OseItemSheet._render (foundry.js:5780)
```

### Foundry VTT Core Version

V10

### Old-School Essentials Core Version

1.6.6

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by wyrmisis** [comment]
This should be fixed by #243 .

**Comment by BuzzardB** [comment]
> This should be fixed by #243 .

Amazing, made those 2 changes and spells are working now. Thanks.
